version.txt

```diff
@@ -1 +1 @@
-v1.0.0
```

Comment:
via release process, gha is in "alpha" and we should reflect the version as such